packages/qdrant-loader/src/qdrant_loader/core/state/state_manager.py

```
@@ -290,6 +383,32 @@ async def get_document_state_record(
+            f"Getting document state records for {source_type}:{source} (batch of {len(document_ids)})"
+        )
+        try:
+            return await _transitions.get_document_state_records_by_ids(
+                self._session_factory,  # type: ignore[arg-type]
+                source_type=source_type,
+                source=source,
+                document_ids=document_ids,
+                project_id=project_id,
+            )
+        except Exception as e:
+            self.logger.error(
```

Ruff (0.15.14)

[warning] Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)